stable/redis-ha/templates/redis-ha-statefulset.yaml

```diff
@@ -147,6 +150,25 @@ spec:
           name: data
         - mountPath: /probes
           name: probes
+{{- if .Values.exporter.enabled }}
+      - name: redis-exporter
+        image: "{{ .Values.exporter.image }}:{{ .Values.exporter.tag }}"
+        imagePullPolicy: {{ .Values.exporter.pullPolicy }}
+        args:
+        {{- range $key, $value := .Values.extraArgs }}
+          - --{{ $key }}={{ $value }}
+        {{- end }}
+        env:
+          - name: REDIS_ADDR
+            value: redis://localhost:6379
+        {{- if .Values.auth }}
+          - name: REDIS_PASSWORD
+            value: {{ .Values.redisPassword }}
+        {{- end }}
```

Comment:
We should allow for resource requests/limits as well. Something like:
```
        resources:
{{ toYaml .Values.exporter.resources | indent 10 }}
```

Reply:
Done